csharp/ql/src/change-notes/2025-10-17-location-in-web-config.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* the `cs/web/missing-x-frame-options` query now correctly handles configuration nested in a `<location >` elements.
```

Comment:
a `<location >` elements. <-- Looks like there is something with singular/plural form.